src/coreclr/inc/corinfo.h

```diff
@@ -317,7 +317,7 @@ struct SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR
     }
 };
 
-// StructFloadFieldInfoFlags: used on LoongArch64 architecture by `getLoongArch64PassStructInRegisterFlags` and
+// StructFloatFieldInfoFlags: used on LoongArch64 architecture by `getLoongArch64PassStructInRegisterFlags` and
 // `getRISCV64PassStructInRegisterFlags` API to convey struct argument passing information.
```

Comment:
getRISCV64PassStructInRegisterFlags API no longer exists after this PR

Reply:
updated